tests/phpunit/api/v3/MailingTest.php

```diff
@@ -288,7 +288,6 @@ public function testMailerPreviewRecipients() {
     $params['mailings']['include'] = array();
     $params['mailings']['exclude'] = array();
     $params['options']['force_rollback'] = 1;
-    $params['api.mailing_job.create'] = 1;
```

Comment:
As per the fix, there is no need to create Job in order to fetch recipient data.